week6.3-ex4.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+FIRST_NAMES = ['avi', 'moshe', 'yaakov']
```

Comment:
Why are they consts and not varaibles inside the if name == main? 
they are not used anywhere else and they dont have .
Also, if they are not changed use tuples